qa/rpc-tests/getblockmerkleroots.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python2
+# Copyright (c) 2014 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+# Exercise the getblockmerkleroots RPC function
+
+from test_framework.test_framework import BitcoinTestFramework
+from decimal import Decimal
+from test_framework.authproxy import JSONRPCException
+from test_framework.util import assert_equal, wait_and_assert_operationid_status
+from test_framework.mc_test.mc_test import *
```

Comment:
I would suggest avoiding unused imports.

```suggestion
from test_framework.mc_test.mc_test import CertTestUtils, generate_random_field_element_hex
```